include/shellstrings.h

```diff
@@ -89,31 +122,80 @@ void shell_input(char* input_file, char* status);
  * Returns:
  *  - None.
  */
-void shell_help();
+void shell_help(bool *color);
 
 /*
- * shell_usage - command line usage instructions
+ * shell_start_interactive - prints out interactive starting message
  *
  * Parameters:
+ *  - file, dictionary, mode names
+ *
+ * Returns:
  *  - None.
+ */
+void shell_start_interactive(char *filename, char *dict, char *md, bool *color);
+
+
+/*
+ * shell_modename - prints out mode name (verbose, quiet or interactive)
+ * 
+ * Accepts flag int
+ *      note 1 is quiet, 2 is batch, 3 is interactive
+ */
+char *shell_modename(int mode);
+
+/*
+ * shell_interactive_line_print: prints line, underlining errors
  *
+ * Parameters:
+ *  - lnum: line number
+ *	- line: string of the line being edited
+ *	- underline: string of underlines, highlighting error words
+ *	- returnflag: indicates if line being parsed is the last line (true if it is the last line)
+ *			in this case, there is a formatting issue in printing out the line that needs to be resolved,
+ *			because the text file that was read does not have a newline character
+ *			at the end of the file.
  * Returns:
  *  - None.
  */
-void shell_usage();
+void shell_interactive_line_print(int lnum, char *line, char *underline, bool returnflag, bool *color);
+
+/*
+ * shell_interactive_replacements
+ *
+ * Parameters:
+ *  - word being replaced
+ *  - list of suggestions
+ *  - flag indicating whether suggestion generation succeeded or not
+ *
```

Comment:
Please update this header to accurately reflect the changed function.

Reply:
updated!